### ISSUE TITLE
[BUG] Kill.Switch: flickering text with default cyclerate.

### ISSUE THREAD

**Issue by atomic83GitHub** [PCSX2/pcsx2#4092]
**Describe the bug**
In the game Kill.Switch, the text in menus flicker with default cyclerate:

https://user-images.githubusercontent.com/26351275/103462414-4d3e7680-4d25-11eb-855f-af3c8dff5582.mp4

Note: +3 cyclerate fixes the glitch ( timing hack or lower cyclerate will make the flickering worse).

**To Reproduce**
Load the game, wait in the menus.

**Expected behavior**
The menu should exhibit no flickering like this:

<img width="885" alt="Capture2" src="https://user-images.githubusercontent.com/26351275/103462442-7828ca80-4d25-11eb-880c-2dd855f7f087.PNG">

**Emulation Settings**
+ 3 cyclerate to fix the flickering.

**System Info (please complete the following information):**
PCSX2 Revision: pcsx2-v1.7.0-dev-817-gc998a51f7-windows-x86
OS: Windows 10
CPU: I9 9900k
GPU: RTX 3070

**Logs and Dumps**
https://drive.google.com/file/d/1mkrHLZ_e4kstazmjBcih3QM1CF4BtUSV/view?usp=sharing

**Comment by gitpro03** [comment]
The behaviour changed, now the menu doesn't disappear but, the background video will stutter badly.

**Comment by MrCK1** [comment]
Proper fix seems to be using EE cache. If you enable the option, the text will stop flickering without changing the cyclerate.

**Comment by Mrlinkwii** [comment]
still an issue was of v1.7.5240 , and as @MrCK1  mentioned the Proper fix seems to be using EE cache

**Comment by ScottNash042** [comment]
has been fixed on using v2.7.19
can be closed

**Comment by refractionpcsx2** [comment]
I'd be surprised, that PR was nothing to do with cycle rate :D

**Comment by Mrlinkwii** [comment]
@ScottNash042  did you test the game without the ee+2 overclock the gamedb has?

**Comment by ScottNash042** [comment]
of course, i did i always delete the gameindex so there is no hacks whatsoever
